ci/jenkinsfiles/common-lib.groovy

```diff
@@ -171,6 +171,23 @@ void getPreviewLogs(String namespace) {
   }
 }
 
+String patchFrontendVersion(String version) {
```

Comment:
Not sure if we need this kind of thing... Guess we can discuss about the real need of changing npm version.